kubernetes/run-unit-tests.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
+
+# It seems that Django tries to import the default settings module even if it's told to use a different one.
+# To work around it we need empty local_settings.py so that importing does not fail.
+echo -n "" > "/srv/http/concent_api/concent_api/settings/local_settings.py"
```

Comment:
After talking about it on Slack, we established that this should not be necessary. Please remove it.